tokio/src/process/mod.rs

```diff
@@ -1164,7 +1164,13 @@ impl Child {
 
     /// Forces the child to exit.
     ///
-    /// This is equivalent to sending a `SIGKILL` on unix platforms.
+    /// This is equivalent to sending a `SIGKILL` on unix platforms
+    /// followed by [`wait(2)`](https://man7.org/linux/man-pages/man2/wait.2.html)
+    /// ([`Child::wait`]).
```

Comment:
Ah I meant a link to `Child::wait` method so users can more easily jump between methods.